dev/manifests/setup_sbt.pp

```diff
@@ -1,10 +1,10 @@
-yumrepo { "sbt yum repo":
-  baseurl => "https://dl.bintray.com/sbt/rpm",
-  enabled => true,
-  gpgcheck => 0,
+exec { 'install_sbt_repo':
+  # Command from the sbt getting started page
+  command => 'curl https://bintray.com/sbt/rpm/rpm | tee /etc/yum.repos.d/bintray-sbt-rpm.repo',
+  path    => '/usr/bin:/usr/sbin:/bin:/usr/local/bin',
 }
 
 package { 'sbt':
-  ensure => installed,
-  require => Yumrepo["sbt yum repo"],
+  ensure  => installed,
+  require => Exec["install_sbt_repo"],
```

Comment:
Just to impart some Puppet knowledge, if both resources are in the same manifest file you can use the `->` operator instead of the long-winded `require => ...`.

For example:
```puppet
exec { 'install_sbt_repo': 
...
}
->
package { 'sbt':
...
}
```